### PR TITLE
feat(web-vitals): Add vertical line markers for web vitals on span view

### PR DESCRIPTION
**Pull request by dashed**
Screen Shot 2020-10-23 at 7 43 11 PM

## TODO

- [x] acceptance test

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/interfaces/spans/measurementsPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                    key={label}
+                    label={label}
+                    left={toPercent(bounds.left || 0)}
+                    onMouseEnter={() => {
+                      hoveringMeasurement(hoverMeasurementName);
+                    }}
+                    onMouseLeave={() => {
+                      notHovering();
+                    }}
+                    onMouseOver={() => {
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/spans/measurementsPanel.tsx:70]: Do you need both mouseEnter and mouseOver?

File: src/sentry/static/sentry/app/components/events/interfaces/spans/utils.tsx [hunk 2]
```diff
@@ -609,3 +616,85 @@ export function isEventFromBrowserJavaScriptSDK(event: SentryTransactionEvent):
...
 // PerformancePaintTiming: Duration is 0 as per https://developer.mozilla.org/en-US/docs/Web/API/PerformancePaintTiming
 export const durationlessBrowserOps = ['mark', 'paint'];
+
+export function getMeasurements(event: SentryTransactionEvent): Map<number, string[]> {
+  if (!event.measurements) {
+    return new Map();
+  }
+
+  const measurements = Object.keys(event.measurements)
+    .filter(name => name.startsWith('mark.'))
```
Comment (markstory) [inline src/sentry/static/sentry/app/components/events/interfaces/spans/utils.tsx:626]: Do all the measures start with `mark.`? Or is this added by the SDK?

Comment (dashed) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/utils.tsx:626]: These are added by the SDK.

The convention is that measurements that begin with `mark.` are assumed to be timestamps for the occurrence of the measurement within the span view.

Comment (markstory) [reply src/sentry/static/sentry/app/components/events/interfaces/spans/utils.tsx:626]: Cool, didn't know that.

**Review by markstory — APPROVED** [review]

**Pull request merged**